multigeiger/transmission.h

```diff
@@ -5,6 +5,13 @@
 #ifndef _TRANSMISSION_H_
 #define _TRANSMISSION_H_
 
+// Sensor-PINS.
+// They are called PIN, because in the first days of Feinstaub sensor they were
+// really the CPU-Pins. Now they are 'virtual' pins to distinguish different sensors.
+// Since we send to sensor.community, we have to use their numbers.
+#define XPIN_RADIATION 19
+#define XPIN_BME280 11
```

Comment:
good to have the explanation here!

if this is specific to sensor.community **only**, the pin stuff should be removed from prepare_http so that that can be used as common code for any json request, without us sending a header that is neither needed nor understood by other servers.